app/services/github_client/user.rb

```diff
@@ -7,10 +7,25 @@ def initialize(username)
     end
 
     def pull_request_events
-      events = (1..10).map do |page|
+      events = retrieve_events
+
+      events.flatten.select { |event| event[:type] == 'PullRequestEvent' }
+    rescue URI::InvalidURIError => exception
+      ExceptionHunter.track(exception)
```

Comment:
Would it make sense to return an empty list instead of `nil` here? (Or whatever Exception Hunter returns)
I think that way we wouldn't need to add [this line](https://github.com/rootstrap/rs-code-review-metrics/pull/344/files#diff-f5e889280495856ba8b2fd80796cc080R9).

Reply:
Yes. It seems like ExceptionHunter returns nil. I thought about doing that too. It might be fine